packages/example/android/app/build.gradle

```diff
@@ -109,6 +112,8 @@ repositories {
 }
 
 dependencies {
+  androidTestImplementation('com.wix:detox:+')
```

Comment:
[question]: Why do we need this?

Reply:
This was added by Faisal for E2E tests, but considering E2E tests for RN will be done via Appium, we don't need this. Removed together with remaining detox configuration.